src/components/Calculator.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import '../App.css';
+
+export default class myComponent extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    return (
+      <div className="calc-container">
+        <div className="calc-buttons">
+          <div className="cell calc-result">0</div>
+          <div className="cell calc-ac">AC</div>
+          <div className="cell calc-plus-minus">+-</div>
+          <div className="cell calc-mode">%</div>
+          <div className="cell calc-division">/</div>
+          <div className="cell calc-seven">7</div>
+          <div className="cell calc-eight">8</div>
+          <div className="cell calc-nine">9</div>
+          <div className="cell calc-multiply">*</div>
+          <div className="cell calc-four">4</div>
+          <div className="cell calc-five">5</div>
+          <div className="cell calc-six">6</div>
```

Comment:
- [ ] Kindly make sure your design match the given one.
- - Your design
![yours](https://user-images.githubusercontent.com/65860678/170227259-8bd65803-f723-44d3-bf85-decf7cd729fc.png)
- Given design
![Screenshot from 2022-05-25 11-01-50](https://user-images.githubusercontent.com/65860678/170227309-1afbd9d5-1618-42dc-9727-d896273c51b8.png)

Reply:
thank you for your feedback,
but what the problem with the mod symbol?

Reply:
Hi! for mod symbol it's ok. sorry for that.